.meta/sinks/aws_s3.toml

```diff
@@ -75,3 +75,80 @@ examples = [
 partition_key = true
 templateable = true
 description = "A prefix to apply to all object key names. This should be used to partition your objects, and it's important to end this value with a `/` if you want this to be the root S3 \"folder\"."
+
+[sinks.aws_s3.options.acl]
+type = "string"
+category = "Object Attributes"
+common = false
+required = false
+examples = [ "authenticated-read" ]
+description = "Canned ACL to apply to the created objects. For more information, see [Canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl)."
+
+[sinks.aws_s3.options.grant_full_control]
+type = "string"
+category = "Object Attributes"
+common = false
+required = false
+description = "Gives the named grantee READ, READ_ACP, and WRITE_ACP permissions on the created objects."
+
+[sinks.aws_s3.options.grant_read]
+type = "string"
+category = "Object Attributes"
+common = false
+required = false
+description = "Allows the named grantee to read the created objects and their metadata."
```

Comment:
Its not completely clear what needs to be set in this field? Maybe an example might be helpful?

Reply:
When creating an object, you can name another entity to have read, write, or full access to the object. This names that other entity. FWIW the wording is almost straight out of the S3 put object docs.